### ISSUE TITLE
Not working Link in Aschaffenburg

### ISSUE THREAD

**Issue by Docmale** [digitalfabrik/integreat-app#2886]
### Describe the Bug
This is a lowest priority bug, since the effect is max hidden and self-solving by additional tapping on same link. Found it on Pixel 3-emulator (android 14) and can reproduce on Samsung A53 (14) where te effect is something different, see steps.

### Steps to Reproduce

1. Open 'Hallo Aschaffenburg' - tile 'Hallo!' (perhaps this only happens in production apps)
2. Swipe up and tap 'Aschaffenburg in Zahlen'
3. Swipe all the way up 
4. Tap at 'Broschüre zum Familienleben' -> page opens; all fine now tap at back
5. Again tap at 'Broschüre zum Familienleben' -> page jumps by about 250 pixel down; 
6a. on emulator: one of the above links is opend
6b. on device: no link opens
7. Again tap at 'Broschüre zum Familienleben' -> page opens; all fine

### Additional Information

I can reproduce this behavior as every second tap on one of the two last links on this page (Bildung, Familienleben) when page is maximum swiped up. It can't be reproduced for the first link (Aschaffenburg in Zahlen) and I havn't seen this effect at any other point.

**Comment by LeandraH** [comment]
@Docmale  Is it possible that it's the same behaviour as in #2160 ?

**Comment by LeandraH** [comment]
Hmm, I just tested it a little and the jumping happens for me only when I tap the first time. In fact, if I go on a new page, it jumps down when I tap somewhere the first time, and if I go back to a previous page, it jumps up on the first tap.

I attached a longer video, is that the same behaviour that you are seeing, or is this a different problem?

[Screen_recording_20240830_122122.webm](https://github.com/user-attachments/assets/e68a3344-52df-4e63-b57d-d15cd54bbe45)
